lib/gui/modules/drive-scanner.js

```diff
@@ -52,7 +53,9 @@ driveScanner.factory('DriveScannerService', (SettingsModel) => {
 
         return drive;
       });
+    })
 
+    .map((drives) => {
```

Comment:
I'm not familiar with the coding style used here (so I can't quite tell what this is doing) but how come the old code had two `map` functions, but the new code has three?